rpc/rpcserver/server.go

```diff
@@ -2004,9 +2010,10 @@ func (s *walletServer) ConfirmationNotifications(svr pb.WalletService_Confirmati
 }
 
 // StartWalletLoaderService starts the WalletLoaderService.
-func StartWalletLoaderService(server *grpc.Server, loader *loader.Loader, activeNet *netparams.Params) {
+func StartWalletLoaderService(server *grpc.Server, loader *loader.Loader, activeNet *netparams.Params, appData string) {
```

Comment:
we shouldn't need to pass around the appdata this way. the appdata can be determined by the db location, which the loader is aware of.